### DISCUSSION TITLE
Olimex poe CMT2300a

### DISCUSSION THREAD

**Discussion by turrican944** [tbnobody/OpenDTU#1283] [Show and tell]
Ich hatte mal wieder etwas lange weile und neulich meine Olimex DTU wiedergefunden. Da ich meine beiden HM600 durch einen HMS1600 ersetzt habe lief das teil natürlich nicht mehr, also mal Adapterplatine gemacht für das CMT2300a Funkmodul auf Olimex POE.
![PXL_20230828_181631488](https://github.com/tbnobody/OpenDTU/assets/70096465/3924c529-a550-4ac6-a3a7-2a77a4e4bdc8)
![PXL_20230828_181136793](https://github.com/tbnobody/OpenDTU/assets/70096465/5bd1277b-2007-4d74-9e9e-eaec66a9531f)
![PXL_20230828_181100274](https://github.com/tbnobody/OpenDTU/assets/70096465/b99ec210-9604-4d9b-81df-5d12615982cf)

**Comment by muellerjm** [comment]
Top! Das ist genau das was ich suche. hast du evtl. noch Platinen über? Oder den Schaltplan... dann lasse ich das selbst über jlcpcb fertigen....

**Comment by muellerjm** [comment]
und evlt. auch die Config?

**Comment by turrican944** [comment]
Moin
Config muss ich aus der platformio.ini bei mir Mal rauskopieren. Platinen habe ich noch liegen , wie viele brauchst du ?
Ich bin aber nur noch bis morgen Abend zu Hause dann ist erstmal Urlaub.
Schaltplan ist als pdf im Git.
https://github.com/turrican944/OpenDTU-PCB/tree/main/PCB/Olimex
Im Schaltplan ist auch das pinning

**Comment by muellerjm** [comment]
mir würde eine genügen... würde gleich mit Paypal Freunde zahlen..
Danke
jm

Am Do., 5. Okt. 2023 um 10:14 Uhr schrieb Florian ***@***.***
>:

> Moin
> Config muss ich aus der platformio.ini bei mir Mal rauskopieren. Platinen
> habe ich noch liegen , wie viele brauchst du ?
> Ich bin aber nur noch bis morgen Abend zu Hause dann ist erstmal Urlaub.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tbnobody/OpenDTU/discussions/1283#discussioncomment-7195261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJFREXRMTB5PKWYAACGCYTDX5ZT6VAVCNFSM6AAAAAA4B2KNFGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCOJVGI3DC>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Mit freundlichen Grüßen

Joachim Müller

**Reply by turrican944** [reply]
Florian(ät)jufu.org
3€ ist ja kompakt und passt in Briefumschlag. Alles andere wäre glaube ich versandtechnisch oversized.

**Comment by turrican944** [comment]
[Device Profil D1 ESP32.json.txt](https://github.com/tbnobody/OpenDTU/files/12829461/Device.Profil.D1.ESP32.json.txt)
Moin
Eintrag für die Platine in die Platformio.ini bzw. platformio_override.ini

[env:flo_cmt2300a_olimex]
; https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware
board = esp32-poe
build_flags = ${env.build_flags}
    -DCMT_CLK=15
    -DCMT_CS=16
    -DCMT_FCS=13
    -DCMT_SDIO=2
;    -DCMT_GPIO2=14
;    -DCMT_GPIO3=5
    -DOPENDTU_ETHERNET

GPIO2 und GPIO3 sind optional und können auf der Platine mit Lötjumpern verbunden werden, geht aber auch ohne. Ich weis ich nicht so genau wofür man die braucht.

Deviceprofil habe ich mit angefügt
https://github.com/turrican944/OpenDTU-PCB

**Reply by hewilf** [reply]
Hallo turrican944, hast Du noch 2 Platinen übrig?
Falls ja, lass mir doch bitte Deine Paypal-Mail zukommen, danke und
viele Grüße
Hewilf

**Comment by MatzeMuc86** [comment]
Welche Variante des ESP hast du genau?
Warum braucht man eine Adapterplatine? Hätte aber ebenfalls Bedarf - sind noch welche übrig?
Hast du einen Link zum Gehäuse?

Danke!

**Comment by turrican944** [comment]


**Reply by MatzeMuc86** [reply]
Cool, bitte kurze Info zwecks Preis und Paypal Freunde via PM. Danke!

**Comment by turrican944** [comment]


**Reply by MatzeMuc86** [reply]
Guten Morgen,
sicher, dass die Nachricht so in der öffentlichen Diskussion stehen soll? ;-).
Grüße

**Comment by labmaster** [comment]
@turrican944 
Hallo,
hast du noch PCB's .
Falls ja würde ich mich über eine Nachricht bezüglich Betrag ... freuen.
VG

**Comment by MasterOfPuppets111** [comment]
Moin, bin auch gerade dabei meinen normalen opendtu esp32 durch einen Olimex Poe zu ersetzen, hättest du noch Platinen da?

Gruß Pierre

**Comment by turrican944** [comment]
Moin

Ich glaube ich habe noch welche, schaue ich heute Abend mal nach

Gruß florian

**Reply by commyonline** [reply]
Hallo Florian, hast du immer noch welche dieser Platinen abzugeben? Ich stelle gerade um auf eine DTU mit dem CMT2300A Chip und hätte gerne noch eine. Viele Grüße Timo

**Comment by commyonline** [comment]
Hallo Florian, hast du immer noch welche dieser Platinen abzugeben? Ich stelle gerade um auf eine DTU mit dem CMT2300A Chip und hätte gerne noch eine. Viele Grüße Timo

**Comment by commyonline** [comment]
Hallo Florian, hast du immer noch welche dieser Platinen abzugeben? Ich stelle gerade um auf eine DTU mit dem CMT2300A Chip und hätte gerne noch eine. Viele Grüße Timo

**Comment by commyonline** [comment]
Okay, das war auffällig... Sorry!

**Comment by ZfT2** [comment]
Danke für den Entwurf der Platine, hatte manges Erfahrung mit dem SMD löten anfangs etwas Mühe aber es funktioniert nun wunderbar.

Aufgrund der Mindestbestellmenge habe ich nun einige Platinen (und Buchsenleisten) über, die ich gerne abgebe. Wer eine braucht kann sich bei mir melden.

**Reply by digit666** [reply]
Hallo ZfT2, ich kann eine PCB für Olimex poe CMT2300a gebrauchen. Hast Du noch eine PCB und eine Buchsenleiste über?

**Reply by ZfT2** [reply]
Hallo, ja kann ich Dir gerne schicken, wie kann ich Dich denn kontaktieren?

**Reply by Keulma01** [reply]
Hallo ZfT2, ich kann eine PCB für Olimex poe CMT2300a gebrauchen. Hast Du noch eine PCB und eine Buchsenleiste über?  Ich lebe in Holland, hoffentlich ist das ok? Paypal ist prima wenn moeglich, Danke, Marc

**Reply by ZfT2** [reply]
Hallo, ja kann ich Dir gerne schicken, sagst Du mit Deine E-Mail Adresse? Dann melde ich mich.

**Comment by Keulma01** [comment]
***@***.*** ist meine email Adresse

Op di 4 nov 2025 om 20:20 schreef ZfT2 ***@***.***>

**Reply by ZfT2** [reply]
Danke, hab Dir eine Mail geschrieben.
